gulp/tasks/styles.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var $ = require( 'gulp-load-plugins' )();
+var pkg = require( '../config' ).pkg;
+var banner = require( '../config' ).banner;
+var config = require( '../config' ).styles;
```

Comment:
Should be `var styles`, see https://nodejs.org/api/child_process.html for example of naming style.